package/gluon-respondd/luasrc/lib/gluon/upgrade/400-respondd-firewall

```diff
@@ -6,26 +6,46 @@ uci:delete('firewall', 'wan_announced')
 
 -- Allow respondd port on WAN to allow resolving neighbours over mesh-on-wan
 uci:section('firewall', 'rule', 'wan_respondd',
-  {
-    name = 'wan_respondd',
-    src = 'wan',
-    src_ip = 'fe80::/64',
-    dest_port = '1001',
-    proto = 'udp',
-    target = 'ACCEPT',
-  }
+{
+	name = 'wan_respondd',
+	src = 'wan',
+	src_ip = 'fe80::/64',
+	dest_port = '1001',
+	proto = 'udp',
+	target = 'ACCEPT',
+}
 )
```

Comment:
is there a style checker that I can use? I'd rather have some lint tool yell at me for not following the conventions instead of taking someone elses time...